StripeConnect/StripeConnect.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,9 @@
 
 /* Begin PBXBuildFile section */
 		41A2A5622C5A972A0077FC74 /* Placeholder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 41A2A5612C5A972A0077FC74 /* Placeholder.swift */; };
+		41A2A5642C5ABD6C0077FC74 /* StripeConnectTests.xctestplan in Resources */ = {isa = PBXBuildFile; fileRef = 41A2A5632C5ABD6C0077FC74 /* StripeConnectTests.xctestplan */; };
+		41A2A5682C5AC5120077FC74 /* StripeCore.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 41A2A5672C5AC5120077FC74 /* StripeCore.framework */; };
+		41D17A452C5A73A6007C6EE6 /* StripeConnect.docc in Sources */ = {isa = PBXBuildFile; fileRef = 41D17A442C5A73A6007C6EE6 /* StripeConnect.docc */; };
```

Comment:
Looks like the .docc got added back to the project file

Reply:
Good catch, weirdly it doesn't actually show in the file navigator. I've got a follow up PR, coming and I'll delete it there.